backend/doc.MD

```diff
@@ -0,0 +1 @@
+ https://blog.appsignal.com/2021/11/10/a-guide-to-load-testing-nodejs-apis-with-artillery.html
```

Comment:
As said before, this should be wrapped in some kind of explanation, no one wants to follow links blindly even if this one is quite explicit.

Maybe this belongs in the wiki part of the project with a step by step/getting started guide?